modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/PortletBase.es.js

```diff
@@ -161,55 +174,8 @@ class PortletBase extends Component {
 	 * @return {Element} The portlet's default root node element.
 	 */
 	rootNodeValueFn_() {
-		return document.getElementById(
-			`p_p_id${this.portletNamespace || this.namespace}`
-		);
+		return document.getElementById(`p_p_id${this.portletNamespace}`);
 	}
 }
 
-/**
- * State definition.
- *
- * @ignore
- * @static
- * @type {!Object}
- */
-PortletBase.STATE = {
-
-	/**
-	 * Portlet's namespace.
-	 *
-	 * @deprecated As of Judson (7.1.x)
-	 * @instance
-	 * @memberof PortletBase
-	 * @type {string}
-	 */
-	namespace: {
-		validator: isString,
-	},
-
-	/**
-	 * Portlet's namespace.
-	 *
-	 * @instance
-	 * @memberof PortletBase
-	 * @type {string}
-	 */
-	portletNamespace: {
-		validator: isString,
-	},
-
-	/**
-	 * Portlet's root node element.
-	 *
-	 * @instance
-	 * @memberof PortletBase
-	 * @type {Element}
-	 */
-	rootNode: {
-		setter: toElementHelper,
-		valueFn: 'rootNodeValueFn_',
```

Comment:
Hey, since you're getting the value from the selector on rootNodeValueFn_ on the component's constructor, why not remove this method from the class?

Reply:
@diegonvs I hadn't thought too much about cleaning up the actual code, I was mostly focused on getting metal removed. I will send an update with this though.